app/javascript/app.js

```diff
@@ -88,10 +88,27 @@ export default template({
     },
     sortStatements: function (statements) {
       return statements.sort((a, b) => {
+        const typeOrder = [
+          'verifiable',
+          'unverifiable',
+          'reconcilable',
+          'actionable',
+          'manually_actionable',
+          'done',
+          'reverted'
```

Comment:
I think I'd want a different order here, because, say, if something's been marked as "unverifiable", there's usually nothing more you'll do with that item, so it should go to the bottom - generally it'd be good to have the statements where we're encouraging someone to do something at the top. @tmtmtmtm should probably check this, but I think I'd go for something like:

`verifiable`,
`reconcilable`,
`actionable`,
`manually_actionable`
`reverted`,
`unverifiable`
`done`,

Reply:
At the bottom, I agree with `reverted`, `unverifiable` and then `done`.

The top is slightly trickier. My initial inclination was to put `manually_actionable` above `verifiable` and `reconciliable`, as resolving the problems at Wikidata will then make them flow into the "later" statuses. But maybe this is because I'm already in "expert mode" rather than "beginner mode" 

Similarly `actionable` is slightly odd: AIUI things should only be there (other than temporarily, whilst they're auto-submitting) if something has gone wrong somewhere. So my initial reaction was to also have that very high: perhaps even at the very top — but, again, I fear that might only really apply to more advanced users.

So I think I support the above ordering by @mhl, though I think we should think about #220 a little harder as well.

Reply:
:+1: I've fixed the order that statements are displayed in 38d6e5b0fe21f0d0d39262480a0e0317423e5b33.